application/models/Question.php

```diff
@@ -1060,9 +1060,16 @@ public function search()
         $criteria->with = [
             'group' => ['alias' => 'g'],
             'questionl10ns' => ['alias' => 'ql10n', 'condition' => "language='" . $this->survey->language . "'"],
-            'question_theme' => ['alias' => 'qt', 'condition' => "extends=''"]
         ];
 
+        // The question theme table holds records for question_themes, which can be:
+        // - user created question_themes,
+        // - LS created core question_themes or
+        // - basic question_types (which can be thought as a basic question themes).
+        // As we want to filter by the description of the question type, we perform a join to the question_type record,
+        // not to the question_theme used.
+        $criteria->join = "JOIN {{question_themes}} as qt ON t." . Yii::app()->db->quoteColumnName("type") . " = qt.question_type AND extends = ''";
```

Comment:
There is already a getQuestionType() method. Already behaves kind of like a relationship and allows to enhance the returning object.

I think would be confusing to add the relationship as it may lead to sometimes use the method and others to use the relationship.

I would just leave it like this for this fix.